sdk/storage/storage-blob/src/PageBlobClient.ts

```diff
@@ -65,6 +72,14 @@ export interface PageBlobCreateOptions {
    * @memberof PageBlobCreateOptions
    */
   metadata?: Metadata;
+  /**
+   * Customer Provided Key Info.
+   *
+   * @type {Models.CpkInfo}
+   * @memberof PageBlobCreateOptions
+   */
+  customerProvidedKey?: Models.CpkInfo;
+  tier?: PremiumPageBlobTier | string;
```

Comment:
Minor: add the @type and @memberof info, I find these are added for track 2, would better to make this consistent if this is convention for track 2.

Reply:
Added